fedn/network/storage/statestore/stores/dto/metric.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from typing import Optional
+
+from fedn.network.storage.statestore.stores.dto.shared import BaseDTO, Field, NodeDTO, PrimaryID
+
+
+class MetricDTO(BaseDTO):
+    metric_id: str = PrimaryID(None)
+
+    key: str = Field(None)
+    value: float = Field(None)
+
+    timestamp: Optional[datetime] = Field(None)
+
+    sender: NodeDTO = Field(NodeDTO())
+
+    model_id: str = Field(None)
+    model_step: int = Field(None)
+
+    round_id: Optional[str] = Field(None)
+    session_id: Optional[str] = Field(None)
+
+
+"""
+log_metric(loss: 1, accarcy: 0.9) # 0
+log_metric(loss: 0.9) # 1
```

Comment:
how would (if possible) the API call look like for logging the metric of a global model?